src/content/en/2021/seo.md

```diff
@@ -1,20 +1,723 @@
+
+{{ figure_markup(
+   image="visible-raw-words-percentile.png",
+   caption="Visible words raw by percentile.",
+   description="Bar chart showing ...",
+   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQ7oSHa6DHYTfZqsuGmIYdlfYVkwrUvOYD_r6soecExV_ZpbbZjmG6watu0hwrOKqK3-inNrt0TfXCO/pubchart?oid=58186900&format=interactive",
+   sheets_gid="455169599",
+   sql_file="seo-stats-by-percentile.sql"
+   )
+}}
+
+Overall, 15% of written content on desktop devices is generated by JavaScript and 14.3% on mobile versions.
+
+### Structured Data
+
+Historically, search engines have worked with unstructured data: the piles of words, paragraphs and other content that comprise the text on a page.
+
+Schema markup and other types of structured data provide search engines another way to parse and organize content. Structured data powers many of [Google's search features](https://developers.google.com/search/docs/advanced/structured-data/search-gallery).
+
+Like words on the page, structured data can be modified with JavaScript.
+
+42.5% of mobile pages and 41.8% of desktop pages have structured data in the HTML. JavaScript modifies the structured data on 4.73% of mobile pages and 4.52% of desktop pages.
+
+On 1.73% of mobile pages and 1.42% of desktop pages structured data is added by JavaScript where it didn't exist in the initial HTML response.
+
+{{ figure_markup(
+   image="structured-data-usage.png",
+   caption="Structure data usage.",
+   description="Bar chart showing ...",
+   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQ7oSHa6DHYTfZqsuGmIYdlfYVkwrUvOYD_r6soecExV_ZpbbZjmG6watu0hwrOKqK3-inNrt0TfXCO/pubchart?oid=1924313131&format=interactive",
+   sheets_gid="1347655296",
+   sql_file="seo-stats.sql"
+   )
+}}
+
+#### Most popular structured data formats
+
+There are several ways to include structured data on a page: JSON-LD, microdata, RDFa, and microformats2. JSON-LD is the most popular implementation method. Over 60% of desktop and mobile pages that have structured data implement it with JSON-LD.
+
+Among websites implementing structured data, over 36% of desktop and mobile pages use microdata and less than 3% of pages use RDFa or microformats2.
+
+{{ figure_markup(
+   image="structured-data-formats.png",
+   caption="Breakdown of structured data formats.",
+   description="Bar chart showing ...",
+   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQ7oSHa6DHYTfZqsuGmIYdlfYVkwrUvOYD_r6soecExV_ZpbbZjmG6watu0hwrOKqK3-inNrt0TfXCO/pubchart?oid=1433352391&format=interactive",
+   sheets_gid="1113852331",
+   sql_file="structured-data-formats.sql"
+   )
+}}
+
+Structured data adoption is up a bit since last year. It's used on 33.17% of pages in 2021 vs 30.60% in 2020.
+
+#### Most popular schema types
+
+The most popular schema types found on homepages are WebSite, SearchAction, WebPage. SearchAction is what powers the Sitelinks Search Box.
```

Comment:
I'm questioning the numbers here. We have website at 29%, searchaction at 25%. Looking at the previous years I think 2019 calculated like we did but 2020 did it differently. Maybe they took 29% of the 33% of pages using structured data? I'm not sure what's correct.

2020:
WebSite has grown 9.37% on desktop and 10.5% on mobile
SearchAction has grown 7.64% on both desktop and mobile
WebPage has grown on desktop 6.83% and 7.09% on mobile
Organization has grown on desktop 4.75% and 4.98% on mobile
ImageObject has grown 6.39% on desktop and 6.13% on mobile

2019:
WebSite (16.02%)
SearchAction (14.35%)
Organization (12.89%)
WebPage (11.58%)
ImageObject (5.35%)

Reply:
cc @tunetheweb @obto

Reply:
The raw data from 2020 is here: https://docs.google.com/spreadsheets/d/1ram47FshAjzvbQVJbAQPgxZN7PPOPCKIK67VJZCo92c/edit#gid=475182211

It shows:

client | type | total | count | pct
-- | -- | -- | -- | --
mobile | schema.org/WebSite | 6,347,640 | 1,611,371 | 25.39%
desktop | schema.org/WebSite | 5,593,642 | 1,475,571 | 26.38%
mobile | schema.org/SearchAction | 6,347,640 | 1,395,979 | 21.99%
desktop | schema.org/SearchAction | 5,593,642 | 1,267,622 | 22.66%
mobile | schema.org/WebPage | 6,347,640 | 1,166,384 | 18.38%
mobile | schema.org/Organization | 6,347,640 | 1,089,686 | 17.17%

So basically took 2020 - 2019 dat to show the growth, rather than what it had grown too.

To be honest I think that's quite confusing (it perhaps would have made more sense if they'd included a chart with he numbers, above, and then discussed what that mean in terms of growth. But they didn't.

So I prefer what you have now (and what 2019 did), as opposed to what last year did, so will leave it as is.